extensions/amp-access/0.1/amp-access.js

```diff
@@ -149,6 +166,9 @@ export class AccessService {
 
     // Start authorization XHR immediately.
     this.runAuthorization_();
+
+    // Wait for the "view" signal.
+    this.scheduleView_();
```

Comment:
Should this be called every time the doc becomes visible? (And short circuit if we have ever counted a view for this render?